Cabal/src/Distribution/Types/ModuleRenaming.hs

```diff
@@ -121,6 +130,11 @@ moduleRenamingParsec bp mn =
         P.spaces -- space isn't strictly required as next is an open paren
         hides <- bp (P.sepBy mn cma)
         return (HidingRenaming hides)
+    parseQualified = do
+        _ <- P.string "qualified"
```

Comment:
This is wrong. It would allow `qualifiedBlabla`

Reply:
some day I will remember that Cabal's parser doesn't actually have a lexer stage before hand  🤣

Reply:
Nor `parsec` has auto-`try`. (But `Described` generative tests should catch that)